tests/cephadm/test_client.py

```diff
@@ -49,10 +49,68 @@ def put_file(client, file_name, content, file_mode, sudo=True):
     nodes_ = config.get("nodes", config.get("node"))
     if nodes_:
         if not isinstance(nodes_, list):
-            nodes = [nodes_]
+            nodes_ = [{nodes_: {}}]
 
         def setup(host):
-            node = get_node_by_id(cls.cluster, host)
+            rhcs_version = str(
+                [host[entry].get("release", "default") for entry in host][0]
+            )
+            name = [entry for entry in host][0]
+            node = get_node_by_id(cls.cluster, name)
+
+            rhel_version = node.distro_info["VERSION_ID"][0]
+            log.debug(
+                f"Passed RHCS version is : {rhcs_version} for host : {node.hostname}\n"
+                f"with RHEL major version as : {rhel_version}"
+            )
+            enable_cmd = "subscription-manager repos --enable="
+            disable_all = [
+                r"subscription-manager repos --disable=*",
+                r"yum-config-manager --disable \*",
+            ]
+            cmd = 'subscription-manager repos --list-enabled | grep -i "Repo ID"'
+            cdn_ceph_repo = {
```

Comment:
I think we need to stop adding the repos in bootstrap or upgrade where it is setting repo for al nodes.